src/tic_tac_toe/input.clj

```diff
@@ -5,13 +5,20 @@
 
 (defn prompt-terminal [question]
    (println question)
-   (read-string(read-line)))
+   (read-line))
 
 (defn valid-selection [input board]
   (and (number? input) (validmove? input) (moveopen? board input)))
 
-(defn user-input-move [board ]
- (let [input (prompt-terminal "what is your next move ?")]
+(defn user-marker []
+  (let [selection  (prompt-terminal "Select your marker x or o ?")]
+    (if  (or (= selection "o") (= selection "x"))
+      selection
+      (do (println "invalid selection")
+      	   user-marker))))
+
+(defn user-input-move [board]
+ (let [input (read-string (prompt-terminal "what is your next move ?"))]
     (if (= true (valid-selection input board))
```

Comment:
If `valid-selection` returns a boolean there is no reason to assert that it equals `true`. Also, I would recommend updating functions that return booleans to end with a question mark, it makes it much easier to understand what a function is doing / returns when it is named properly.

https://github.com/bbatsov/clojure-style-guide#pred-with-question-mark